rt/features/logging/pom.xml

```diff
@@ -38,6 +38,10 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
```

Comment:
maybe a bit overkill for the actual usage? IMHO it would be great to drop or reuse cxf utils.

Reply:
Perhaps, I will check

Reply:
Agreed, there is no need to import it just for StringUtils.containsIgnoreCase

Reply:
updated